components/kyma-environment-broker/internal/broker/plans_validator_test.go

```diff
@@ -13,20 +13,10 @@ func TestNewPlansSchemaValidatorErrors(t *testing.T) {
 		inputJSON    string
 		expErr       string
 	}{
-		"missing name, not valid components list": {
-			againstPlans: []string{GCPPlanID, AzurePlanID},
-			inputJSON:    `{"components": ["wrong component name"]}`,
-			expErr:       `(root): name is required, components.0: components.0 must be one of the following: "kiali", "tracing"`,
-		},
-		"missing name, not valid machine type": {
```

Comment:
should we remove this test?

Reply:
we shouldn't - I have reverted tests for regions and machine types 👍